core/services/chainlink/application.go

```diff
@@ -103,6 +104,8 @@ type Application interface {
 	ID() uuid.UUID
 
 	SecretGenerator() SecretGenerator
+
+	EALRequestRouter() eal.RequestRouter
```

Comment:
Why does this have to be part of the `Application` interface? Is there a way where we don't have to expose routing here? I'm not sure it belongs in this layer.

Reply:
Also I don't see where this is used - seems like we create it from `eallib` but doesn't seem to be used in the CL app.

Reply:
oops sent out a new revision where this method is used in web/router.go. It's debatable whether this belongs in this layer but I followed existing convention [here](https://github.com/smartcontractkit/chainlink/blob/develop/core/web/pipeline_runs_controller.go#L119). We're using webhook job runner from application in the web controller.